bonfire/namespaces.py

```diff
@@ -339,3 +341,68 @@ def extend_namespace(namespace, duration, local=True):
         raise FatalError("Reservation lookup failed")
 
     log.info("reservation for ns '%s' extended by '%s'", namespace, duration)
+
+
+def describe_namespace(project_name: str):
+    output = ""
+    if not project_exists(project_name):
+        return f"Error no namespace called {project_name} was found"
+
+    if not project_name.startswith("ephemeral") or project_name == "default":
+        output += "Can't get project info. Please use an ephemeral oc project\n"
+        output += "Hint: run 'oc project <ephemeral-namespace>' and retry\n"
+        output += "If you do not know your namespace, use 'bonfire namespace list'\n"
+        return output
+    routes = oc("get", "route", "-o", "json", _silent=True).strip()
+    host = get_fe_hostname(routes)
+
+    kc_name = f"env-{project_name}-keycloak"
+    fe_creds = get_keycloak_creds(oc("get", "secret", kc_name, "-o", "json", _silent=True).strip())
+    output += f"Current project: {project_name}\n"
+    project_url = get_console_url()
+    if project_url:
+        ns_url = f"{project_url}/k8s/cluster/projects/{project_name}"
+        output += f"Namespace console url: {ns_url}\n"
+    output += f"Frontend route: https://{host}\n"
+    output += f"Keycloak login: {fe_creds['username']} | {fe_creds['password']}\n"
+    return output
+
+
+def project_exists(project_name: str):
```

Comment:
You can remove this func if you make the changes I suggested within the `describe_` func